nep-22.mediawiki

```diff
@@ -0,0 +1,71 @@
+<pre>
+  NEP: 22
+  Title: Contract Update Function
+  Author: Owen Zhang <zhangzhihong@ngd.neo.org>, Fernando Díaz Toledano <shargon@gmail.com>, Erik Zhang <erik@neo.org>, Roman Khimov <roman@nspcc.ru>
+  Type: Standard
+  Status: Accepted
```

Comment:
This standard is accepted, shall we update the README.md?

Reply:
I think we better merge this, #182 and then update the README, because there are some omissions there anyway.

Reply:
The scheme is exactly to avoid endless reapprovals, the document is fine now and that's the main thing here.